### PR TITLE
Wercker pipeline deploys to releases

### PR DESCRIPTION
**Pull request by nicolaifsf**
Pipeline for wercker and sd build to deploy to releases

### CODE REVIEW

File: wercker.yml [hunk 1]
```diff
@@ -64,3 +64,24 @@ deploy:
       hostname: github.com
       fingerprint: 16:27:ac:a5:76:28:2d:36:63:1b:56:4d:eb:df:a6:48
       type: rsa
+
+    - script:
+        name: get highest version from tags
+        code: export VERSION=$(bash incrementVersion.sh)
```
Comment (stjohnjohnson) [inline wercker.yml]: Can you use gitversion instead?  You can download it like we do here: https://github.com/screwdriver-cd/job-tools/blob/master/Dockerfile#L17-L20

File: wercker.yml [hunk 2]
```diff
@@ -64,3 +64,31 @@ deploy:
...
       fingerprint: 16:27:ac:a5:76:28:2d:36:63:1b:56:4d:eb:df:a6:48
       type: rsa
+
+    - script:
+        name: download git version
+        code: |
+          wget $(wget -q -O - https://api.github.com/repos/screwdriver-cd/gitversion/releases/latest \
+            | awk '/browser_download_url/ { print $2 }' \
+            | sed 's/"//g') \
+            && chmod +x gitversion \"')
```
Comment (stjohnjohnson) [inline wercker.yml]: I think this \"') is wrong.

**Comment by tkyi** [comment]
👍  after squashing

File: wercker.yml [hunk 3]
```diff
@@ -64,3 +64,31 @@ deploy:
...
       fingerprint: 16:27:ac:a5:76:28:2d:36:63:1b:56:4d:eb:df:a6:48
       type: rsa
+
+    - script:
+        name: download git version
+        code: |
+          wget $(wget -q -O - https://api.github.com/repos/screwdriver-cd/gitversion/releases/latest \
+            | awk '/browser_download_url/ { print $2 }' \
+            | sed 's/"//g') \
+            && chmod +x gitversion \
```
Comment (stjohnjohnson) [inline wercker.yml]: No need for the extra `\`

File: VERSION [hunk 4]
```diff
@@ -0,0 +1 @@
+1.0.0
```
Comment (stjohnjohnson) [inline VERSION]: Not needed anymore.

**Comment by jer** [comment]
👍

**Pull request merged**